### ISSUE TITLE
Homebridge-Homematic zeigt die komplette Wetterstation nicht

### ISSUE THREAD

**Issue by Ace-li-k** [thkl/homebridge-homematic#580]
Hallo und danke vorab für das Plugin.

Ich habe diese Wetterstation für Homematic nachgebaut.
[https://github.com/jp112sdl/HB-UNI-Sen-WEA/wiki](url)

Nur leider wird mir in HOMEKIT nur die Temperatur und Feuchte angezeigt.

Meine Config besteht derzeit nur aus diesem Inhalt.

`"platforms": [
       {
           "platform": "HomeMatic",
           "name": "HomeMatic CCU",
           "ccu_ip": "**IPADRESSECCU**",
           "subsection":"Homekit"
       }   
   ]`

Wie könnte man diese erweitern, damit alles angezeigt wird?
Bin etwas Ratlos.

Vielen Dank

**Comment by dkrizic** [comment]
Siehe hier https://github.com/thkl/homebridge-homematic/issues/103

Es gibt in HomeKit sowas wie eine Wetterstation nicht.

**Comment by thkl** [comment]
Man kann das schon machen. In Eve werden die Characteristics such angezeigt nur in  Home ned ... ich schau mir das mal an. Focus liegt aktuell aber beim  hap-Homematic Projekt

**Comment by Ace-li-k** [comment]
Vielen Dank für die schnelle Antwort. 

Aber selbst in der EVE app habe ich nur diese beiden Typen... bzw muss ich ein anderes gerät hinzufügen als Homebridge / hoobs!?

BZW was ist das hap-Homematic Projekt?  Homematic Acces Point?

Es gibt ja noch die Apps Homey oder Home+ sähe es da anders aus?

**Comment by thkl** [comment]
hap-Homematic : HomeKit direkt auf der CCu3/RaspberryMatic 

Für die Implementierung des wettersensors bräuchte ich die Infos über die Kanäle und Datenpunkte von dem Gerät.

**Comment by Ace-li-k** [comment]
ah cool.

Wenn du mir sagen kannst was genau wo zu finden ist stehe ich gern zum testen und liefern zur Verfügung

**Comment by thkl** [comment]
In meinen Repos :)

**Comment by Ace-li-k** [comment]
Ich bin leider kein Homematic Fachmann und auch Github ist wirklich neuland für mich. Bitte das etwas zu berücksichtigen. 
Mit Kanäle meinst du wie das Gerät heisst ? Seriennummer bezeichnung?
Das ganze steckt natürlich in dem HOMEKIT Gewerke für Homekit

**Comment by Ace-li-k** [comment]
Oder brauchst du sogar eine Datei von der CCU?

**Comment by thkl** [comment]
ne das bekomme ich schon aus der Software raus

**Comment by thkl** [comment]
> Ich bin leider kein Homematic Fachmann und auch Github ist wirklich neuland für mich. Bitte das etwas zu berücksichtigen.
https://github.com/thkl/hap-homematic

**Comment by Ace-li-k** [comment]
ach, dann hab ich dich damit falsch verstanden.  Du Hattest auf HAP-Homematic verwiesen.  Danke

Dennoch, wenn fragen sind oder sonstiges , ich stehe dafür gern zur Verfügung.

**Comment by thkl** [comment]
So, ich hab mir das mal angeschaut. Es gibt einen fertigen Service dafür.
In der config.json muss das hier bei services nachgetragen werden:

```
{
 "type": "HB-UNI-Sen-WEA:WEATHER",
 "service": "HomeMaticHomeKitWeatherStationService"
}

```

Wenn es noch kein services Eintrag gibt, würde das so ausschauen:


```
"services": [
	{
 "type": "HB-UNI-Sen-WEA:WEATHER",
 "service": "HomeMaticHomeKitWeatherStationService"
}]
```

**Comment by Ace-li-k** [comment]
Moin, Danke fürs schauen.

also meine Config.json schaut nun so aus. 
Passt das so? 

`{
            "platform": "HomeMatic",
            "name": "HomeMatic CCU",
            "ccu_ip": "192.168.178.28",
            "filter_channel": [
                "BidCos-RF.KEQXXXXXXX:4",
                "BidCos-RF.LEQXXXXXXX:2"
            ],
            "outlets": [
                "BidCos-RF.KEQXXXXXXX:4",
                "BidCos-RF.IEQXXXXXXX:1"
            ],
            "enable_hmip": "true",
            "variables": [
                "pooltemp"
            ],
            "services": [
                {
                    "type": "HB-UNI-Sen-WEA:WEATHER",
                    "service": "HomeMaticHomeKitWeatherStationService"
                }
            ],
            "subsection": "Homekit",
            "plugin_map": {
                "plugin_name": "homebridge-homematic"
            }
        },`

**Comment by Ace-li-k** [comment]
Leider das einzige was dazu gekommen ist, ist der LUX wert. 
Allerdings mit 0,0001

**Comment by thkl** [comment]
ja, die Apple Home App kennt keine Wetterstation, daher wird da nix angezeigt.
Zum LuX .. welchen Wert liefert denn der Sensor ? Lux oder pseudo Lux ??

**Comment by Ace-li-k** [comment]
Was meisnt du mit Pseudo ?

Es ist der MAX44009 Sensor verbaut. in der CCU2 steht aktuell  :  Helligkeit   800.60 LUX

**Comment by thkl** [comment]
Stimmt, der Datenpunkt im Sensor heisst LUX und nicht BRIGHTNESS wie in der originalen Wetterstation ..

**Comment by Ace-li-k** [comment]
passt meine confi sonst?
kann man das "beheben"?

**Comment by thkl** [comment]
ja und ja (neues ServiceFile bauen)

**Comment by Ace-li-k** [comment]
Super Danke.

Mache ich das oder musst du das machen?

**Comment by thkl** [comment]
ich muss es bauen, du musst es einbinden

**Comment by Ace-li-k** [comment]
Ok, Danke für die bisherigen Antworten und Hilfestellungen.

Dann bin ich gespannt und warte einfach mal . 
Also,. ob du es überhaupt vorhast .

Dennoch Danke

**Comment by Ace-li-k** [comment]
Moin,

Ich möchte gar kein Stress aufkommen lassen.
Wollt nur mal Fragen wie lange so etwas dauern könnte?
Gruß und danke

**Comment by thkl** [comment]
> Ich möchte gar kein Stress aufkommen lassen.

Sehr gut

> Wollt nur mal Fragen wie lange so etwas dauern könnte?

Und warum machst Du das jetzt dann doch ?

**Comment by Ace-li-k** [comment]
Moin

:)

Gibt es schon etwas neues ?!

Vielen Dank schönen Start in die Woche

**Comment by Ace-li-k** [comment]
@thkl moin

Wirst du das noch weiter verfolgen oder haben andere Projekte Vorrang ?

Gruß und dank

**Comment by Ace-li-k** [comment]
@thkl 
Darf ich nochmal fragen ob noch etwas geplant ist ? Nicht das ich umsonst hoffe ;) 

Danke
